modules/user_manual/pages/session_management.adoc

```diff
@@ -1,36 +1,19 @@
-= Manage Connected Browsers and Devices
+= Session Management
 
-The personal settings page allows you to have an overview on the
-connected browsers and devices.
+The personal settings page allows you to have an overview of the connected browsers and clients.
 
-[[managing-connected-browsers]]
-== Managing Connected Browsers
+image:settings_sessions.png[Sessions]
 
-In the list of connected browsers you see which browsers connected to
-your account recently:
+[[sessions]]
+== Sessions
 
-image:settings_sessions.png[List of browser sessions.]
+In the list of sessions you see which browsers and clients are connected to your account recently. You can use the trash icon to terminate any of the sessions in the list. If you want to lock out a user, you would need to change his password and then terminate his session. You can not terminate the session you are using to access the sessions page.
 
-You can use the trash icon to disconnect any of the browsers in the
-list.
+[[app-passwords]]
+== App Passswords
 
-[[managing-devices]]
-== Managing Devices
+At the bottom of the list you find a button to create a new device-specific password. You can choose a name to identify the token later. The generated password is used for configuring the new client. Ideally, generate individual tokens for every device you connect to your account, so you can disconnect those individually if necessary.
```

Comment:
After experimenting with this functionality, this paragraph isn't quite accurate. Based on what I've found, here's how I would rewrite it:

```asciidoc
Underneath the "_App passwords / tokens_" list is a button to create a new app or device-specific username and password. The username will be pre-filled to the currently logged in user, and a random password will be generated. You can change both of these, if so desired. When you're happy with the username and password, click the btn:[done] button.

IMPORTANT: Once the password / token is created, it cannot be changed; it can only be deleted.

TIP: We recommend that you generate tokens for every device you want to connect to your ownCloud instance, as this will allow you to disconnect connections individually, if necessary.
```

Reply:
Sorry, again, about the formatting, but I'm hoping to save you time and effort.